sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -48,8 +50,7 @@ namespace Azure { namespace Core {
    * @remark This class is supposed to be able to handle a DateTime that comes over the wire.
    */
   class DateTime : public Details::Clock::time_point {
-    static DateTime const SystemClockEpoch;
-
+    AZ_CORE_EXPORT static DateTime const SystemClockEpoch;
```

Comment:
Do we really need this static `DateTime` or is there a way to re-write the implementation to avoid it? What's its purpose?

Reply:
We can avoid, but I see no reason.
With inline functions it is a bit faster. Maybe it is not a bottleneck, but anyways, we can avoid it in this place, but htere are dozens more, especially in the Extensible Enums area, where there is no escape, and this problem must be solved.